### ISSUE TITLE
Add Australian AAQ NEPM AQI algorithm

### ISSUE THREAD

**Issue by libokj** [hrbonz/python-aqi#21]
![image](https://user-images.githubusercontent.com/66983285/110891973-20629d00-832e-11eb-823d-8a9d380c96e4.png)

![image](https://user-images.githubusercontent.com/66983285/110892099-5d2e9400-832e-11eb-9655-c8cbaf8fcd6d.png)

NO2, O3, SO2: 1-hour average;
CO: 8-hour average;
PM10, PM2.5: 24-hour average

Sources:
https://soe.environment.gov.au/theme/ambient-air-quality/topic/2016/air-quality-index
https://soe.environment.gov.au/theme/ambient-air-quality/topic/2016/national-air-quality-standards

**Comment by libokj** [comment]
https://www.der.wa.gov.au/your-environment/air/air-quality-index

It seems that Western Australia has one more level beyond **Very Poor**: 200+ for **Extreme**.

**Comment by hrbonz** [comment]
This looks like a very straightforward index:
* If PM2.5 is 25µg/m³ over 24h then AQI is 100
* if PM2.5 is 5µg/m³ over 24h then AQI is 20
* if PM2.5 is 100µg/m³ over 24 then AQI is 400

Looks like there's no global AQI which avoids comparing numbers that have nothing to do with each other.

**Comment by libokj** [comment]
> * if PM2.5 is 5µg/m³ over 24h then AQI is 20

Yes, the concentration is directly proportional to the AQI and vice versa.

AQI = measurement concentration / maximum concentration * 100
measurement concentration = AQI / 100 * maximum concentration

So we can simply use the formula for conversion.
